config/config.py

```diff
@@ -142,6 +148,13 @@ def all_src_exclusions(self):
                                          self.excluded_src_file_names,
                                          self.excluded_src_file_extensions)
 
+    @property
+    def version_property_prefix_suffix(self):
+        """
+        Convenience method that returns a tuple of prefix and suffix to use for generated version properties.
+        """
+        return (self.version_property_prefix, self.version_property_suffix)
+
```

Comment:
This is fine, but I would just have 2 properties.... that's just less weird.

Reply:
Will do! Yeah, I was kind of hesitated between keeping less parameters passed around and making the property type clear